package.json

```diff
@@ -6,9 +6,10 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint build/",
-    "build": "npm run build-core && npm run build-validation",
+    "build": "npm run build-core && npm run build-validation build-propertyDependencies",
     "build-core": "node build/build.js < jsonschema-core.md > jsonschema-core.html",
-    "build-validation": "node build/build.js < jsonschema-validation.md > jsonschema-validation.html"
+    "build-validation": "node build/build.js < jsonschema-validation.md > jsonschema-validation.html",
+    "build-propertyDependencies": "node build/build.js < proposals/propertyDependencies.md > proposals/propertyDependencies.html"
```

Comment:
I think I'd prefer to have a `build-proposals` script instead of one for each, but I'm too lazy right now to figure out how to do that.

Reply:
I pushed an update to handle builds in a more scalable way. Now you can run `npm run build-all` to build everything and `npm run build -- propsosal/propertyDependencies.md` to build a specific file. You can also use wildcards to build multiple files.